test/Hpack/ConfigSpec.hs

```diff
@@ -991,7 +991,7 @@ spec = do
               foo:
                 ain: driver/Main.hs
             |]
-          readPackageConfig file `shouldReturn` Left (file ++ ": Error in $.executables.foo: failed to parse field executables: The key \"main\" was not found")
+          (`shouldSatisfy` isLeft) =<< readPackageConfig file
```

Comment:
Minor bikeshedding, I think
```
readPackageConfig file >>= (`shouldSatisfy` isLeft)
```
reads nicer, as it is closer to a proper English sentence.
```